src/granite_orm/error.cr

```diff
@@ -1,7 +1,7 @@
 class Granite::ORM::Error
   property field, message
 
-  def initialize(@field : Symbol, @message : String)
+  def initialize(@field : (String | Symbol), @message : String?)
```

Comment:
Does this effect how model error messages are rendered? Maybe it should default to “” instead of being nilable?